setup.py

```diff
@@ -70,6 +84,22 @@ def build_extensions(self):
     if not os.path.exists(join("pyopenjtalk", "openjtalk" + ext)):
         raise RuntimeError("Cython is required to generate C++ code")
 
+# make openmp available
+system = platform.system()
+if system == "Windows":
+    extra_compile_args = []
+    extra_link_args = ["/openmp"]
+elif system == "Linux":
+    extra_compile_args = ["-fopenmp"]
+    extra_link_args = ["-fopenmp"]
+elif system == "Darwin":
+    os.system("brew install libomp")
```

Comment:
I don't think automatically running `brew install` in setup.py is a good idea. How about updating the installation guide instead?

Reply:
Actually the parallel is optional. Without linking to openmp, cython's prange will fallback to a single thread loop.

Reply:
brew has merely become the default package manager in MacOS, thus in 99% case, installing openmp means run the command `brew install libomp`. However, we can just remove this line and give a warning of falling back to single thread loop to user.

Reply:
Giving a warning sounds good to me. It is better to avoid installing binary dependencies to `/usr/local/bin` or somewhere implicitly.